.github/workflows/ci.yml

```diff
@@ -106,6 +106,8 @@ jobs:
 
       - name: Run Barge
         working-directory: ${{ github.workspace }}/barge
+        env:
+          AQUARIUS_VERSION: v3.1.1
```

Comment:
ideally we have this as default in barge and not hardcode specific versions in here before merging

Reply:
yes, once this is merged, we will update barge and remove it
If we update barge, all ocean.js tests will fail now